src/frontend/github-oauth.ts

```diff
@@ -91,12 +92,16 @@ export default (opts: OAuthOptions): GithubOAuth => {
       );
 
       req.session.githubToken = response.data.access_token;
+
       if (!req.session.githubToken) {
         return next(new Error('Missing Access Token from Github OAuth Flow.'));
       }
-      return res.redirect(redirectUrl);
+
+      return res.redirect(redirectUrl.toString());
     } catch (e) {
       return next(new Error('Cannot retrieve access token from Github'));
+    } finally {
+      elapsedTimeMetrics(`https://${host}/login/oauth/access_token`);
```

Comment:
Not sure what does it do here.

elapsedTimeMetrics will return a function which won't be executed anywhere. 

If we want to measure the endpoint time we should add it as a middleware function to the endpoint itself, not here.

Also we shouldn't use `host` as part of tags parameter. If we'll have too many hosts we'll explode tags cardinality (We should keep variants of each tag value low, otherwise it will cause problems in SignalFX)